src/Sulu/Bundle/MediaBundle/Resources/public/scss/modules/collection.scss

```diff
@@ -42,74 +45,181 @@
         margin-bottom: -20px;
         position: relative;
         text-align: center;
+    }
 
-        .masonry-item {
-            width: 190px;
-            background-color: $white;
-            border-radius: 3px;
-            opacity: 1;
-            @include transition(opacity 100ms, top 100ms, left 100ms);
-
-            &.loading {
-                opacity: 0;
-            }
+    .masonry-item {
+        $width: 260px;
+        $checkboxWidth: 15px;
+        $checkboxMargin: 8px;
+        box-shadow: 0 0 10px -10px rgba(0, 0, 0, 0.2);
+
+        width: $width;
+        background-color: $white;
+        border-radius: $borderRadius;
+        opacity: 1;
+        border: 1px solid $white;
+        @include transition(
+            opacity $fadeInDuration,
+            top $fadeInDuration,
+            left $fadeInDuration,
+            box-shadow $effectDuration
+        );
+
+        &.loading {
+            opacity: 0;
+        }
 
-            .masonry-head {
-                background-color: $grayLighterer;
-                width: 100%;
+        &.action-navigator {
```

Comment:
If I remember right, it's in the media-selection where the whole masonry item is an action navigator. This was not introduced by this PR. But I'll have a look at it. Maybe we can get rid of this case.